### DISCUSSION TITLE
BirdWeather Support eMail

### DISCUSSION THREAD

**Discussion by timsterc** [mcguirepr89/BirdNET-Pi#1147] [General]
Hey Everyone,

In order to keep better track of BirdNET-Pi requests - I'd like to request that going forward they be sent to birdnetpi@birdweather.com

It would also be wonderful if it is possible to modify the BirdWeather ID Request form to direct to that email as well.

Thanks!
-Tim
Founder - BirdWeather

**Comment by Svardsten53** [comment]
Hi Tim,

Unfortunately there are no more BirdNET-Pi updates, but hopefully someone can make this change to the general setup.

In the next few weeks you are probably going to have some new request from Sweden. I'm not able to do the change in the github installation setup, but have done a manual change in the memoy card cloned images the new users are going to use after personal modifications in the settings.

**Comment by tsiawd** [comment]
I manully sent a request to the new email can I use the original template as a layout and just copy paste?
